docs/rules/jest/no-if.md

```diff
@@ -0,0 +1,34 @@
+# Prevent if statements in tests. (jest/no-if)
+This rule prevents the use of if statements in tests. 
+
+## Rule Details
+
+If statments in tests is usually an indication that a test is attempting to cover too much. Each branch of code executing within an if statement will likely be better served by a test devoted to it.
```

Comment:
Not only covering too much, but also possibly not actually running the test you think it is